### ISSUE TITLE
Deno.emit() source maps: sources are wrapped inside `<>`

### ISSUE THREAD

**Issue by bxantus** [denoland/deno#10953]
I am using Deno for client side web development (with preact) and I've tried to bundle my sources using `Deno.emit()`, so I can  have the source maps too, besides the bundle to ease debugging.
This almost works, but all the entries inside the `"sources"` array are wrapped inside angle brackets, like this:
```json
"sources":["<file:///C:/work/travelmate/src/map/placeDescriptions.ts>",
                  "<file:///C:/work/travelmate/src/map/placeData.ts>", ...
```
I'm using the Debugger for Firefox extension in vscode, and unfortunately it can't resolve these URLs correctly, because of the brackets (if I remove them everything is ok). Firefox debugger is more tolerant, but it won't remove the trailing `>` characters from file names...

I couldn't find anything about the format of source URLs in in the specs. Is there any particular reason to wrap these URLs in brackets?
If no, I would suggest to remove this wrapping and output the URLs in simple encoded string form.

> NOTE: (related info) I'm using `Deno.emit()` as it can output source maps too, in contrast with `deno bundle`.  I've seen
> that  when #4499 will be solved, then deno bundle could output source maps too. Would you consider to use the
> same output as `Deno.emit()`? This would be welcome, as without source maps debugging bundles (for client side web apps)
> can be tedious. In my opinion Deno is a great tool for easily creating client side web apps in typescript, only the deployment needs a bit of attention.

**Comment by kitsonk** [comment]
> Is there any particular reason to wrap these URLs in brackets?

It is because the transpiler we use (swc) does when "custom" module names are used (which are used when we pass them to swc) and we don't "clean" them up.

**Comment by bxantus** [comment]
It is fixed with Deno 1.7.1 (probably in the newer swc)
